SPECS/ruby/ruby.spec

```diff
@@ -124,6 +121,8 @@ Provides:       rubygem-base64 = %{base64_version}-%{release}
 Provides:       rubygem(base64) = %{base64_version}-%{release}
 Provides:       rubygem-benchmark = %{benchmark_version}-%{release}
 Provides:       rubygem(benchmark) = %{benchmark_version}-%{release}
+Provides:       rubygen-bundler = %{bundler_version}-%{release}
+Provides:       rubygen(bundler) = %{bundler_version}-%{release}
```

Comment:
We should also add `Obsoletes: rubygem-bundler < %{bundler_version}-%{release}` and remove the separate `rubygem-bundler.spec`. Without the `Obsoletes` anyone stuck on the separate package (current with lower version and we won't be updating it, since we have the built-in here) won't be able to properly update, because package managers won't recognize this package as an update path.

Reply:
https://github.com/microsoft/CBL-Mariner/pull/4452